New Bank/NewBank/newbank/server/NewBank.java

```diff
@@ -53,9 +62,24 @@ public synchronized String processRequest(CustomerID customer, String request) {
 				String to = input[3];
 				return moveFunds(customer,amount,from,to);
 			}
+			if(input[0].equals("PAYMENT")) {
+				if(input.length < 4) { // return fail if not enough information is provided
+					return "FAIL123";
+				}
+				Customer custCredit = customers.get(customer.getKey());
+				double amount = Double.parseDouble(input[1]);
+				String fromAccount = input[2];
+				String toAccount = input[3];
+				String debitBank = input[4];
+				String credtiBank = input[5];
+				Customer custDebit = customers.get(customerId);
```

Comment:
If updating parameters of a command, also update the protocol.txt file to outline what each parameters are. In this sprint, we had planned to implement only the requirements already given to us in the protocol.txt file. i.e PAY <Person> <amount>.